tools/dockerfile/interoptest/grpc_interop_http2/build_interop.sh

```diff
@@ -27,5 +27,7 @@ ${name}')
 cp -r /var/local/jenkins/service_account $HOME || true
 
 # compile the tests
-(cd /var/local/git/grpc/tools/http2_interop && go test -c)
+(cd /var/local/git/grpc/tools/http2_interop && \
```

Comment:
Actually, to be future proof, let's create a `go.mod` file instead?

```sh
cd tools/http2_interop
go mod init
go mod tidy
```

Reply:
Done.